sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -83,10 +101,27 @@ def get_sku_df() -> pd.DataFrame:
     )
     print(f'Done fetching SKUs')
     items = json.loads(proc.stdout.decode('ascii'))
-    df = pd.DataFrame(items)
+    new_items = []
+    for item in items:
+        zones = item['locationInfo'][0]['zones']
+        region = item['locations'][0]
+        if region not in REGION_SET:
+            continue
+        if len(zones) == 0:
```

Comment:
Would you add a comment that explains in which case this happens?

Reply:
TODO for myself: remove the AZ column in azure.csv

Reply:
Done. I removed the AZ for azures for now, as we are not using them.